app/views/amusement_parks/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1><%= @amusement_park.name %></h1>
+<h3>Admission cost: <%= @amusement_park.admission_cost %></h3>
+
+<h2>Park Mechanics</h2>
+<section class='mechanics'>
+  <ol>
+    <% @amusement_park.unique_name.each do |mechanic| %>
+    <li>
+      <h3>Mechanic name: <%= mechanic %></h3>
+    </li>
+    <%end%>
+  </ol>
+</section>
```

Comment:
In your block variable I would make it more descriptive, it looks like you're pulling the entire `mechanic` object just looking at the name. I would call it `mechanic_name` or something like that to make it obvious in your code. I was ready to say that you weren't pulling out the name on line 9 with `mechanic.name` before having to go back to the model and see that you were using `pluck(:name)` which is fine but make the block variable representative of that.